fluentd/logger/log.rb

```diff
@@ -0,0 +1,384 @@
+#
```

Comment:
Lets move this into the fluentd/patch/logger directory to better identify what it is